### PR TITLE
Initial commit for dpdk container

### PR DESCRIPTION
**Pull request by krharsh**

### CODE REVIEW

File: containers/agent/dpdk/vrouter-dpdk/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    apt-get install -y contrail-vrouter-agent contrail-vrouter-utils contrail-vrouter-dpdk && \
+    apt-get clean && \
+    rm -rf /var/cache/apt && \
+    ldconfig
+
+COPY 10-core-pattern.conf /etc/sysctl.d/
+COPY entrypoint.sh /
+
+ENTRYPOINT ["/entrypoint.sh"]
+CMD ["/usr/bin/contrail-vrouter-agent", "/usr/bin/contrail-vrouter-dpdk", "--conf_file", "/etc/contrail/contrail-vrouter-dpdk.ini"]
```
Comment (Andrey-mp) [inline containers/agent/dpdk/vrouter-dpdk/Dockerfile]: looks like the first arg ("/usr/bin/contrail-vrouter-agent") is not needed

**Comment by Andrey-mp** [comment]
Hi,

1) Please move it all to one level upper to provide more consistent naming:
containers/agent/dpdk/vrouter-dpdk -> containers/agent/vrouter-dpdk
containers/agent/dpdk/vrouter-init-kernel-dpdk -> containers/agent/vrouter-init-kernel-dpdk

2) Please make two Dockerfile for easch new container - for ubuntu and centos (see latest commits for neighbor folders)

File: containers/agent/dpdk/vrouter-dpdk/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+ARG CONTRAIL_REGISTRY=localhost:5000
+ARG CONTRAIL_CONTAINER_TAG=4.1.0.0-6-ubuntu16-newton
+FROM ${CONTRAIL_REGISTRY}/contrail-base:${CONTRAIL_CONTAINER_TAG}
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get install -y contrail-vrouter-agent contrail-vrouter-utils contrail-vrouter-dpdk && \
```
Comment (Andrey-mp) [inline containers/agent/dpdk/vrouter-dpdk/Dockerfile]: please add 'sudo python-opencontrail-vrouter-netns python-contrail-vrouter-api' to packets list as it was done for vrouter

File: containers/agent/dpdk/vrouter-init-kernel-dpdk/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+ARG CONTRAIL_VERSION=4.1.0.0-6
```
Comment (Andrey-mp) [inline containers/agent/dpdk/vrouter-init-kernel-dpdk/Dockerfile]: please remain header in new format (see other containers)

**Comment by Andrey-mp** [comment]
and please move it to gerrit - https://review.opencontrail.org/#/admin/projects/Juniper/contrail-container-builder

this repo has been closed for direct pushes.

**Comment by krharsh** [comment]
All comments incorporated and moved to Gerrit here https://review.opencontrail.org/#/c/38937/
please review

**Comment by Andrey-mp** [comment]
please close this

**Pull request closed without merging**